embulk-output-bigquery.gemspec

```diff
@@ -1,6 +1,6 @@
 Gem::Specification.new do |spec|
   spec.name          = "embulk-output-bigquery"
-  spec.version       = "0.7.2"
+  spec.version       = "0.6.9.trocco.0.1.0"
```

Comment:
based on [embulk/embulk-output-bigquery v0.7.2](https://github.com/embulk/embulk-output-bigquery/releases/tag/v0.7.2), but trocco-io/embulk-output-bigquery supports embulk version 0.9.
so, prefixed it with 0.6.9 (supports embulk 0.9).